internal/rbd/nodeserver.go

```diff
@@ -318,7 +319,7 @@ func (ns *NodeServer) NodeStageVolume(
 	}
 
 	isStaticVol := parseBoolOption(ctx, req.GetVolumeContext(), staticVol, false)
-	rv, err := populateRbdVol(ctx, req, cr)
+	rv, err := populateRbdVol(ctx, req, cr, ns.Driver.CrushLocationMap)
```

Comment:
`populateRbdVol()` could be a function of `NodeServer`, so that you do not need to pass extra arguments.